app/assets/javascripts/books/read_more.js

```diff
@@ -0,0 +1,3 @@
+readMore = function(full_description) {
```

Comment:
@b-artem This function is defined in global scope. Let's isolate it like you did for other pages

Reply:
Done